packages/framework/src/Resources/config/services.yml

```diff
@@ -1006,6 +1006,10 @@ services:
         arguments:
             - '%shopsys.elasticsearch.structure_dir%'
             - '%env(ELASTIC_SEARCH_INDEX_PREFIX)%'
+        calls:
```

Comment:
is this backward compatible?
see the similar incompatibility after merge of https://github.com/shopsys/shopsys/pull/1158/files